plugins/lando-recipes/rails/rails.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * LAMP recipe builder
+ *
+ * @name lamp
```

Comment:
more refactoring required, s/b "rails" not "lamp", throughout

Reply:
Absolutely true, though as indicated, this isn't intended to be done ;) There is far more work to do than that 😀

Reply:
Yup, I'm of the "start simple" school of thought. I'm working on a PR to your PR. I have no idea how to do this: "To play with this, you need to publish a copy of the container somewhere your docker daemon can get to it." But I'll dive in and see if I can figure it out.

Reply:
Oh, is this it? https://github.com/lando/lando/pull/602/files#diff-aad8a5410cee10fcbb10eb81fecd8762R3

Reply:
yep! Exactly, that should work